README.md

```diff
@@ -1,2 +1,299 @@
+        * ICP: `wso2/wso2-integration-control-plane:1.0.0`
+    
+    Example for MySQL:
+
+    ```yaml
+        FROM <base-image>
+        USER root
+        RUN apt-get update && \
+            apt-get install -y wget && \
+            apt-get clean && \
+            rm -rf /var/lib/apt/lists/*
+        ENV WSO2_SERVER_HOME=/home/wso2carbon/wso2-integration-control-plane-1.0.0
+        RUN mkdir -p ${WSO2_SERVER_HOME}/lib
+        ARG JDBC_DRIVER_URL=https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-j-8.0.33.tar.gz
+        RUN wget -O /tmp/mysql-connector.tar.gz "${JDBC_DRIVER_URL}" && \
+            tar -xzf /tmp/mysql-connector.tar.gz -C /tmp && \
+            find /tmp -name "mysql-connector*.jar" -exec cp {} ${WSO2_SERVER_HOME}/lib/ \; && \
+            rm -rf /tmp/mysql-connector.tar.gz
+        RUN chown -R wso2carbon:wso2 ${WSO2_SERVER_HOME}/lib && \
+            chmod -R 755 ${WSO2_SERVER_HOME}/lib
+        USER wso2carbon
+    ```
+ 2. Build the image:
+    ```yaml
+    docker build -t customized-wso2-img:1.0.0 .
+    ```    
+ 3. Update the following configurations in `values_local.yaml`:
+    ```yaml
+    containerRegistry: ""
+    wso2:
+        deployment:
+            image:
+                repository: "customized-wso2-img"
+                tag: "1.0.0"
+                pullPolicy: IfNotPresent
+    ```   
+
+> **Note**: For integration development, if you are using the [MI VSCode extension](https://marketplace.visualstudio.com/items/?itemName=WSO2.micro-integrator), you can add JDBC drivers by placing the JAR file in the `<PROJECT_DIR>/deployment/libs` folder and then clicking "Create Docker Image" under Deployment Options. This will build the Docker image and push it to the local registry. 
+
+<div style="display: flex; justify-content: space-around; align-items: center;">
+    <div style="display: flex; justify-content: space-around; align-items: center;">
+        <figure style="width: 40%; height: 500px; text-align: center;">
+            <img src="resources/project-structure.png" alt="Project Structure" />
+            <figcaption>Add JDBC JAR to '/deployment/libs' folder</figcaption>
+        </figure>
+        <figure style="width: 40%; height: 500px; text-align: center;">
+            <img src="resources/create-docker-image.png" alt="Create Docker Image"/>
+            <figcaption>Click Create Docker Image Button to build integration docker image</figcaption>
+        </figure>
+    </div>
+</div>
+
+### 5. Deploy MI and ICP
+
+#### Deploy MI
+Navigate to the `mi` directory and deploy MI using Helm:
+```bash
+helm install wso2-mi ./mi -f values_local.yaml --namespace wso2-integration
+```
+
+#### Deploy ICP
+Navigate to the `icp` directory and deploy ICP using Helm:
+```bash
+helm install wso2-icp ./icp -f values_local.yaml --namespace wso2-integration
+```
+
+### 6. Verify the Deployment
+
+#### Check Pods
+Ensure all pods are running:
+```bash
+kubectl get pods -n wso2-integration
```

Comment:
Shall we add a sample output as well?

Reply:
Added by https://github.com/wso2/helm-mi/pull/18/commits/f37598283fbeab7a7bc68d258c2ae71777127143